omnibus/config/software/python3.rb

```diff
@@ -1,6 +1,6 @@
 name "python3"
 
-default_version "3.11.5"
+default_version "3.11.8"
```

Comment:
This requires a checksum change on line 18 for Linux / macOS. On Windows, we need to upload the bundle to the `dd-agent-omnibus` bucket first.

Reply:
thanks for the catch with L18! 

Windows bundle is in progress and I'm in touch with Julien in case I get blocked.